using-fetch/exercise.js

```diff
@@ -16,11 +16,14 @@ Expected result
 Open index.html in your browser. Every time you refresh the page,
 a different greeting should be displayed in the box.
 */
+let greetingEl =document.querySelector('#greeting-text')
+fetch("https://www.greetingsapi.com/random")
+  .then(response=>response.json())
+  .then(data =>{console.log(data);
+    greetingEl.textContent = `${data.greeting} That's greeting in ${data.language}`})
```

Comment:
Yes - the API isn't working just now, but I can see that this would work! :rocket: